test/test_things.js

```diff
@@ -0,0 +1,32 @@
+import should     from 'should';
+import supertest  from 'supertest';
+
+import app        from './server';
+import sensorthings    from '../src/sensorthings';
+
+import {
+  BAD_REQUEST,
+  errnos,
+  ERRNO_FORBIDDEN,
+  ERRNO_INVALID_API_CLIENT_NAME,
+  ERRNO_UNAUTHORIZED,
+  errors,
+  FORBIDDEN,
+  UNAUTHORIZED
+} from '../src/errors';
+
+const endpointPrefix = '/api';
+const server = supertest.agent(app);
+let token;
+
+describe('Things API', () => {
+  it('should exist', done => {
+    server.get(endpointPrefix + '/Things')
+      .expect('Content-type', /json/)
+      .expect(200)
+      .end((err, res) => {
+        res.status.should.be.equal(200);
+        done();
+      });
+  });
+});
```

Comment:
Same comment about unused imports from `../src/errors` and unused `token` in all test files.